index.js

```diff
@@ -173,7 +174,7 @@ const getSettings = function(options) {
     testInput: 'test/**/*.test.js',
     distName: basicName,
     exportName: basicName.replace(/-(\w)/g, (matches, letter) => letter.toUpperCase()),
-    browserslist: pkg.browserslist || ['defaults', 'ie 11'],
+    browserslist: pkg.browserslist,
```

Comment:
is it worth it to do `pkg.browerlist || babelConfig.presets[0][1].targets` or something like that? or do the changes in lines 124-126 basically take care of it?

Reply:
Just double checked. There's no need to set a default here if `pkg.browerlist` is undefined because we will just fall back to the default list from `@videojs/babel-config` in that case, which is what we want. See https://github.com/videojs/babel-config/pull/1